src/TransactionController.ts

```diff
@@ -354,30 +361,31 @@ export class TransactionController extends BaseController<TransactionConfig, Tra
 	}
 
 	/**
-	 * Resiliently checks all submitted transactions for confirmations in all
-	 * blocks maintained in a sibling BlockHistoryController
+	 * Resiliently checks all submitted transactions on the blockchain
+	 * and verifies that it has been included in a block
+	 * when that happens, the tx status is updated to confirmed
 	 *
 	 * @returns - Promise resolving when this operation completes
 	 */
 	async queryTransactionStatuses() {
```

Comment:
this is not a PR change but we may want to maintain same async function documentation in the same way. Maybe `async queryTransactionStatuses(): Promise<any>` ? With this I noticed that we have different ways to document it.